.github/workflows/build.yml

```diff
@@ -119,3 +120,119 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           file: ./lcov.info
+
+  package:
+    name: Package
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, macOS-latest, ubuntu-latest]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2019-11-04
+          override: true
+      - name: Install wasm-pack
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: wasm-pack    
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Build Enso IDE
+        run: python script/build.py
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+      - run: npm install
+        working-directory: ./app
+      - run: npm run dist
+        working-directory: ./app
+      - run: ls -R app/dist 
+
+      - uses: actions/upload-artifact@v1
+        name: Upload Artifacts (Ubuntu, AppImage)
+        with:
+          name: Enso IDE (Linux, AppImage)
+          path: app/dist/enso-ide-0.1.0.AppImage
+        if: matrix.os == 'ubuntu-latest'
+
+      - uses: actions/upload-artifact@v1
+        name: Upload Artifacts (Ubuntu, unpacked)
+        with:
+          name: Enso IDE (Linux, unpacked)
```

Comment:
If I understand correctly, "unpacked" means that it was packed in the past and is not packed anymore (https://dictionary.cambridge.org/dictionary/english/unpacked). If I'm right, this is incorrect usage of the word. The correct would be "not packed".